src/main/java/LinksFromSinglePage.java

```diff
@@ -0,0 +1,48 @@
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+public class LinksFromSinglePage<T> implements Links<T> {
```

Comment:
@PavelNejinskiy lets remove generic from here, this impl works with `String` only

Reply:
@PavelNejinskiy `StringLinks`